packages/gestalt/src/contexts/ColorSchemeProvider.js

```diff
@@ -129,20 +129,37 @@ const getTheme = (colorScheme: ?ColorScheme) =>
     ? darkModeTheme
     : lightModeTheme;
 
+type Props = {|
+  /**
+   *
+   */
+  children: Node,
+  /**
+   * The color scheme for components inside the ColorSchemeProvider. Use 'userPreference' to allow the end user to specify the color scheme via their browser settings, using the 'prefers-color-scheme' media query. See [color scheme](https://gestalt.pinterest.systems/colorschemeprovider#Custom-navigation-context) variant for examples.
+   */
+  colorScheme?: ColorScheme,
+  /**
+   * An optional id for your color scheme provider. If not passed in, settings will be applied as globally as possible (ex. setting color scheme variables at :root).
+   */
+  id?: ?string,
+|};
+
 /**
- * [ColorSchemeProvider](https://gestalt.pinterest.systems/colorschemeprovider) is an optional [React context provider](https://reactjs.org/docs/context.html#contextprovider) to enable dark mode
+ * [ColorSchemeProvider](https://gestalt.pinterest.systems/colorschemeprovider) is an optional [React context provider](https://reactjs.org/docs/context.html#contextprovider) to enable dark mode.
  */
 export default function ColorSchemeProvider({
   children,
-  colorScheme,
+  colorScheme = 'light',
```

Comment:
We've been effectively applying light mode as the default in the logic of `getTheme`. This change makes that default explicit.

We'd also previously been stating that light mode was the default in the docs — this change is necessary to continue providing that info with autogenerated docs.